docs/reference_guide.md

```diff
@@ -216,7 +219,10 @@ TRACEPOINT_PROBE(random, urandom_read) {
 }
 ```
 
-This instruments the random:urandom_read tracepoint, and prints the tracepoint argument ```got_bits```.
+This instruments the tracepoint `random:urandom_read tracepoint`, and prints the tracepoint argument ```got_bits```.
+The name of the probe function is `tracepoint__random__urandom_read`.
+This tracepoint probe can be attached with:
```

Comment:
You can make it explicit that explicit tracepoint attachment is only needed for C++? For exmaple, like below
```
For C++, this tracepoint probe can be attached with:
...
```

Reply:
Done, in addition to mentioning C++, I added a statement on the expectation when using the Python API.